packages/react-sdk/src/useChannel.ts

```diff
@@ -0,0 +1,38 @@
+'use client'
+
+import { useContext, useMemo } from 'react'
+import { useCurrentSpaceId, useSpace } from './useSpace'
+import { ChannelContext } from './internals/ChannelContext'
+
+/**
+ *  Views a channel by its channelId and spaceId.
+ */
+export const useChannel = (channelId: string, spaceId: string) => {
+    const space = useSpace(spaceId)
+    const channel = useMemo(() => space.getChannel(channelId), [space, channelId])
+    return channel
```

Comment:
This should be return useObservable(channel) right?

Reply:
Yeah, same from the useSpace discussion: https://github.com/river-build/river/pull/326#discussion_r1662653577